packages/ui/Marquee/MarqueeDecorator.js

```diff
@@ -1,6 +1,6 @@
 /* global ResizeObserver */
 
-import direction from 'direction';
+import {isRtlText} from '@enact/i18n/util';
```

Comment:
This import statement is not been sorted alphabetically.

Reply:
OK